strawberry/field.py

```diff
@@ -85,6 +87,29 @@ def __getattr__(self, attr):
     def __call__(self, *args, **kwargs):
         return self._wrapped_obj(self, *args, **kwargs)
 
+    def _get_permissions(self):
+        """
+        Gets all permissions defined in the permission classes
+        >>> strawberry.field(permission_classes=[IsAuthenticated])
+        """
+        if self.permission_classes:
+            return [permission() for permission in self.permission_classes]
```

Comment:
if we default permission_classes to an empty list we can remove this if
and I'd suggest to change this to a generator so it is only evaluated when using it :)

```python
return (permission() for permission in self.permission_classes)
```

Reply:
Awesome! - that sounds like a good plan, it will be bring in the concept of `lazy` loading which beneficial to this scenario